src/client/visualizations/table/table.scss

```diff
@@ -91,7 +90,7 @@ $header-padding-top: 12px;
       background-color: $white;
       left: 0;
       top: 0;
-      width: $segment-width - $space-left;
+      width: calc(100% - #{$space-left});
```

Comment:
How about browser support?

Reply:
Calc is available in all modern browsers (See [mdn](https://developer.mozilla.org/de/docs/Web/CSS/calc#Browser_Kompatibilit%C3%A4t)). If you prefer I can set the width in JS, doesn't really matter to me 😸

Reply:
Didn't see that before, but it is already used in the code base. See https://github.com/allegro/turnilo/search?q=calc